lib/downloader.js

```diff
@@ -65,9 +65,15 @@ function download (torrentId) {
   var opts = {store: IdbChunkStore}
   webtorrent.add(torrentId, opts, function (torrent) {
     if (torrent.urlList.length === 0) {
-      var swUrl = navigator.serviceWorker.controller.scriptURL
-      torrent.addWebSeed(swUrl.substring(0, swUrl.lastIndexOf('/')))
+      const isSingleFile = torrent.files.length === 1
+      const swUrl = navigator.serviceWorker.controller.scriptURL
+
+      let webSeedUrl = swUrl.substring(0, swUrl.lastIndexOf('/'))
+      if (isSingleFile) webSeedUrl += '/planktos/' + torrent.files[0].path
```

Comment:
It's probably better to use `name` instead of `path` even though in this specific case they are identical.

See the 'info dictionary' section in [bep3](http://www.bittorrent.org/beps/bep_0003.html)